client/my-sites/stats/stats-post-anniversaries/index.jsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { localize, moment } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import ErrorPanel from '../stats-error';
+import StatsList from '../stats-list';
+import StatsListLegend from '../stats-list/legend';
+import SectionHeader from 'components/section-header';
+import StatsContentText from '../stats-module/content-text';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import Card from 'components/card';
+import QueryPosts from 'components/data/query-posts';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSitePostsForQuery, isRequestingSitePostsForQuery } from 'state/posts/selectors';
+
+// Maximum number of simultaneous queries to fetch the posts by year
+const MAX_POST_QUERIES = 10;
+
+const StatModulePostAnniversaries = props => {
+	const { oldestPostQuery, postsByYearQueries, postsByYear, requesting, siteId, translate } = props;
+
+	const allPosts = Array.prototype.concat( ...postsByYear );
+
+	const cardClasses = classNames( 'stats-module', 'is-expanded', 'summary', {
+		'is-loading': requesting,
+		'has-no-data': ! allPosts.length,
+		'is-showing-error': ! allPosts.length,
+	} );
+
+	return (
+		<div>
+			<SectionHeader
+				label={ translate( 'Anniversaries', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Happy Anniversary!' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?